docs/user/gcp/tested_instance_types_arm.md

```diff
@@ -0,0 +1 @@
+* `T2A`
```

Comment:
Suggest to name as the same as [Google doc](https://cloud.google.com/compute/docs/general-purpose-machines#t2a_machines), i.e. `Tau T2A`, thanks!

Reply:
Just updated, thank you!